core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractGroupResolver.java

```diff
@@ -90,4 +90,17 @@ public boolean listGroups(Condition condition, int limit, int offset, String sor
 
         return true;
     }
+
+    @Override
+    public boolean addGroup(Group group, UserStoreManager userStoreManager)
+            throws UserStoreException {
+
+        return true;
+    }
+
+    @Override
+    public boolean deleteGroupByName(String groupName, UserStoreManager userStoreManager) throws UserStoreException {
```

Comment:
why we dont have the deleteGroups using the group id?

Reply:
This is the group resolver. It does not require a delete by id. This will be used mainly to clean SCIM side